source/Willow-Playground-TestRunner/WPTestRunnerToolbar.class.st

```diff
@@ -69,12 +69,13 @@ WPTestRunnerToolbar >> onRunSelectedTestsTrigger [
 WPTestRunnerToolbar >> renderContentOn: html [
 
 	html div
-		class: BootstrapCssStyles buttonToolbar;
-		with: [ html div
-				class: BootstrapCssStyles buttonGroup;
+		with: [ 
```

Comment:
In order to improve readability we should use `apply:[];with:`
otherwise the nesting makes it harder to understand.